### DISCUSSION TITLE
Нет пункта меню "Буду смотреть"

### DISCUSSION THREAD

**Discussion by VyacheslavTeplyakov** [quarckster/kodi.kino.pub#158] [Q&A] [answered]
Пропал несколько версий назад.
Обновил сейчас KODI до 18.0, чистая установка и аддон поставил последний самый через репо. Без изменений.
Что делать?

**Comment by quarckster** [comment]
В настройках плагина включен этот пункт меню?

**Reply by VyacheslavTeplyakov** [reply]
Наверно коряво объяснил. Я про контекстное меню, когда правой мышью щелкаешь по сериалу. Избранное работает. Закладки работают. А все что т касается сериалов, буду смотреть, не буду смотреть, этих пунктов нет.

**Reply by quarckster** [reply]
Это баг, но не думаю, что буду его чинить для Kodi18, теперь все усилия направлены на подержку Kodi19

**Reply by VyacheslavTeplyakov** [reply]
Так и у меня 19.
Опечатался. =)
Странно, что до меня никто не заметил, оно так тыщу лет уже.

**Reply by quarckster** [reply]
Я создал баг #159

**Comment by quarckster** [comment, marked as answer]
В новой версии ошибка исправлена.

**Reply by VyacheslavTeplyakov** [reply]
теперь помер вообще весь плугин, не помогает ни сброс ни переустановка. Лог:
2021-01-27 20:12:12.125 T:336      INFO <general>: initializing python engine.
2021-01-27 20:12:12.428 T:336     ERROR <general>: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                                    - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                                   Error Type: <class 'ModuleNotFoundError'>
                                                   Error Contents: No module named 'm3u8'
                                                   Traceback (most recent call last):
                                                     File "C:\Users\HTPC\AppData\Roaming\Kodi\addons\video.kino.pub\addon.py", line 2, in <module>
                                                       from resources.lib.main import plugin
                                                     File "C:\Users\HTPC\AppData\Roaming\Kodi\addons\video.kino.pub\resources\lib\main.py", line 10, in <module>
                                                       from resources.lib.plugin import Plugin
                                                     File "C:\Users\HTPC\AppData\Roaming\Kodi\addons\video.kino.pub\resources\lib\plugin.py", line 12, in <module>
                                                       from resources.lib.auth import Auth
                                                     File "C:\Users\HTPC\AppData\Roaming\Kodi\addons\video.kino.pub\resources\lib\auth.py", line 14, in <module>
                                                       from resources.lib.utils import notice
                                                     File "C:\Users\HTPC\AppData\Roaming\Kodi\addons\video.kino.pub\resources\lib\utils.py", line 3, in <module>
                                                       import m3u8
                                                   ModuleNotFoundError: No module named 'm3u8'
                                                   -->End of Python script error report<--

**Reply by quarckster** [reply]
У вас должен быть включён стандартный репозиторий Kodi, оттуда должен установиться модуль m3u8 https://kodi.wiki/view/Add-on:M3u8

**Reply by VyacheslavTeplyakov** [reply]
Kodi новый из коробки с официального сайта. Репо включён. Видимо что-то не поставилось само. Попробую принудительно руками.

**Reply by quarckster** [reply]
@VyacheslavTeplyakov если вас не затруднит, то поделитесь результатами.

**Reply by VyacheslavTeplyakov** [reply]
Как ни странно, в вики про него есть, а в репо его нет. Потому и не ставится. Где найти не нашёл. На гитхабе только исходники и я не знаю как их упаковывать в алдоны.

**Reply by quarckster** [reply]
В репе он есть, просто этот как зависимость ставится. Я проверил у себя на Kodi19. Удалил kino.pub и удалил m3u8, установил плагин заново и m3u8 установился автоматически. На всякий случай вот все официальные плагины https://mirrors.kodi.tv/addons/matrix/

**Reply by VyacheslavTeplyakov** [reply]
Чудеса. У меня даже поиском он не находится

**Reply by quarckster** [reply]
Поиском и у меня не находится, Kodi не показывает модули и другие служебные плагины, но скачивает их из репозитория.

**Reply by VyacheslavTeplyakov** [reply]
пришлось еще руками поставить script.module.iso8601
папки стали открываться, но с input stream adaptive вообще ничего не производит, валится в ошибку. а без него, вроде что-то начинает, моргает экраном и виснет. Попробую все зачистить и откатиться на 18ю версию.

**Reply by quarckster** [reply]
Ваш Kodi не устанавливает зависимости, в этом проблема.

**Reply by VyacheslavTeplyakov** [reply]
Да понять бы почему. Старый 18й я удалил, потом папку профиля для верности почистил. Поставил новый 19й и ничего с ним не делал, только язык на русский переключил. Единственное дополнение в нем после установки это кинопаб.
Но в общем проблема с зависанием решилась, поковырял настройки видео и все взлетело на 19й версии.
Спасибо за помощь! =)
19 RC версия всё же, могут быть индивидуальные баги, видимо мне досталось.

**Reply by quarckster** [reply]
@VyacheslavTeplyakov мне кажется, я разобрался с ошибкой, всё-таки это у меня репозиторий был неправильно настроен. Можете попробовать с Kodi19 ещё раз.

**Reply by VyacheslavTeplyakov** [reply]
Да. Сделал по новой чистую установку kodi, добавил репо, поставил из него. Все взлетело с первого раза, без проблем в этот раз. Спасибо!

**Reply by Dimasua** [reply]
> добавил репо

А как Вы добавляете репозиторий?
И Вы ставите kodi отдельно поверх ОС или уже собранные имиджи используете?

**Reply by quarckster** [reply]
> А как Вы добавляете репозиторий?

Написано в README https://github.com/quarckster/kodi.kino.pub#kodi-add-on-repository.

> И Вы ставите kodi отдельно поверх ОС или уже собранные имиджи используете?

Я использую osmc https://osmc.tv/download/

**Reply by Dimasua** [reply]
> Я использую osmc https://osmc.tv/download/

Попробовал и с osmc, но там идет Kodi 18.9, они еще не выпустили релиз с Kodi 19. Поэтому там по прежнему ругается на Python 3.0.0.

**Reply by quarckster** [reply]
Так и ставьте старую версию плагина https://github.com/quarckster/kodi.kino.pub/releases/tag/2.12.4

**Comment by Dimasua** [comment]
Спасибо! Буду пробовать.

**Comment by Dimasua** [comment]
В общем ситуация разрешилась следующим образом: на сегодняшний только в
одной сборке оказывается можно выбрать Kodi 19, это LibreELEC. В остальных
идёт только 18.9. Поэтому выбрав 19ю версию все установилось и взлетело с
первого раза :) Спасибо за помощь и наведении на цель.
